packages/api/src/schema/schema.yaml

```diff
@@ -732,6 +738,15 @@ components:
         response:
           $ref: "#/components/schemas/webhook-response"
 
+    playback-policy:
```

Comment:
WDYT of making this an object with a `type` field instead that has this `enum`? Then it could easily evolve soon to define an actual token-gating spec with some definition like
```json
"playbackPolicy": {
  "type": "tokenGated",
  "erc20": {
    "chain": "polygon",
    "contract": "0x123",
    "minBalance": 100
  }
}
```
or sth like that (token-gating schema is not the point, only having this be an object from the start).

## Devil's advocate 

The counter argument that I give to myself is that we can also not commit the same mistake of over-engineering these schemas and can just start simple instead. Maybe we'll never really implement token-gating fully on our side =P

We can always use a `oneOf` schema when we want to update this field to become an object instead, although that would still be a breaking change as it can explode client implementations out there, especially on static languages that try to parse it as a `string` and suddenly start getting an object.

So anyway, I'm not strongly opinionated on the object in order to KISS. Feel free to just ignore this idea but raising it here for consideration 😁

Reply:
So, are you proposing to make something like
```
"playbackPolicy":{
     "type":"signed"
}```
for now, where `type` is the enum and we can expand later? Sounds good to me, on first iteration when I was putting the signingKey ids inside the policy it was actually like that. wdyt of the object like this?

Reply:
Looks great!